lib/pages/conversation_page.dart

```diff
@@ -0,0 +1,158 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart';
+import 'package:lend_a_hand/services/auth_service.dart';
+
+class Conversation extends StatefulWidget {
+  final FirebaseUser firebaseUser;
+  final String reciever;
+
+  const Conversation({Key key, this.firebaseUser, this.reciever}) : super(key: key);
+
+  @override
+  _ConversationState createState() => _ConversationState();
+}
+
+class _ConversationState extends State<Conversation> {
+  final FirebaseAuth _auth = FirebaseAuth.instance;
```

Comment:
You should use the `AuthProvider` here.

Reply:
It is actually not being used currently in code lol